jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/http/DslCookieManager.java

```diff
@@ -15,10 +15,41 @@
  * This element has to be added before any http sampler to be considered, and if you add multiple
  * instances of cookie manager to a test plan, only the first one will be considered.
  *
+ * Clearing cookies on each iteration is defaulted to true but setClearCookiesBetweenIterations()
+ * can be used to set to false if required.
+ * The default cookie policy is 'standard' but setCookiePolicy() can be used to specify the
+ * required cookie policy
+ *
  * @since 0.17
  */
 public class DslCookieManager extends AutoEnabledHttpConfigElement {
 
+  protected String cookiePolicy;
+  protected boolean clearEachIteration = true;
+
+  public enum CookiePolicy {
```

Comment:
Can you add javadocs to cookie policy values so users understand when makes sense to use each one?

Reply:
added